README.md

```diff
@@ -1,21 +1,161 @@
 # Python Matter Server
 
-This project implements a Matter Controller Server using WebSockets and provides both a server and client implementation.
-The goal of this project is primary to Matter support for Home Assistant but the unioversal approach makes it suitable to be used in other projects too.
+This project implements a Matter Controller Server over WebSockets using the [official CHIP SDK](https://github.com/project-chip/connectedhomeip) as a base and provides both a server and client implementation.
 
-This repository is for development only. For enabling Matter support within Home Assistant, please refer to the Home Assistant documentation.
+The goal of this project is primary to have Matter support in Home Assistant but its universal approach makes it suitable to be used in other projects too.
 
-:warning: This project is under development and considered BETA !
+This repository is for development only (so not for enduser support). For enabling Matter support within Home Assistant, please refer to the [Home Assistant documentation](https://www.home-assistant.io/integrations/matter/).
+
+NOTE: Both Matter and this implementation are in early (v1) state and probably features are missing or could be improved. See our [project board](https://github.com/orgs/home-assistant-libs/projects/1) for status updates and maybe something you'd like to help out with development and/or testing.
```

Comment:
```suggestion
NOTE: Both Matter and this implementation are in early (v1) state and features are probably missing or could be improved. See our [project board](https://github.com/orgs/home-assistant-libs/projects/1) for status updates and maybe something you'd like to help out with, development and/or testing.
```

Reply:
Maybe link to the development section since we write the same there?

Reply:
something like this? (I've edited your text)

Reply:
Great!